DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Raw/BraceToSpeckleConverter.cs

```diff
@@ -0,0 +1,47 @@
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+// POC: There is no validation on this converter to prevent conversion from "not a Revit Beam" to a Speckle Beam.
+// This will definitely explode if we tried. Goes back to the `CanConvert` functionality conversation.
+// As-is, what we are saying is that it can take "any Family Instance" and turn it into a Speckle.RevitBeam, which is far from correct.
+// CNX-9312
+public class BraceToSpeckleConverter : IRawConversion<DB.FamilyInstance, SOBR.RevitBrace>
+{
+  private readonly IRawConversion<DB.FamilyInstance, SOBR.RevitBeam> _beamConverter;
+
+  public BraceToSpeckleConverter(IRawConversion<DB.FamilyInstance, SOBR.RevitBeam> beamConverter)
+  {
+    _beamConverter = beamConverter;
+  }
```

Comment:
Hmm not sure if the beam and the brace should be built one on top of each other, I guess we're missing some mid-level class to join the dots between them.